CONTRIBUTING.md

```diff
@@ -1,65 +1,231 @@
-visit the [Flutter community](https://flutter.dev/community) page.
-
-# Ways to contribute
-
-We encourage you to reach out and join the conversation.
-
-An easy way to send feedback is to "thumbs up" issues important to you
-in either the issue tracker for the [Flutter SDK and API docs][issues],
-or the [flutter.dev website][doc-issues].
-
-Other ways you can contribute:
-
-* [Ask HOW-TO questions that can be answered with specific solutions][so]
-* [Live chat with Flutter engineers and users][chat]
-* [Discuss Flutter, best practices, app design, and more on our
-   mailing list][mailinglist]
-* [Report bugs and request features][issues]
-* [Report API docs bugs][issues]
-* [Submit PRs to the Flutter SDK][PRs]
-* [Request docs for flutter.dev][doc-issues]
-* [Submit PRs to flutter.dev][doc-PRs]
-* [Follow us on Twitter: @flutterdev](https://twitter.com/flutterdev/)
-* [Read the Flutter Publication on Medium](https://medium.com/flutter)
-* [Sign up to Future UX Studies on Flutter](https://flutter.dev/research-signup)
-* [Join the subreddit to keep up with the latest in the Flutter
-   community][reddit]
-* [Join the Discord to connect with other developers
-   and discuss Flutter][discord]
-
-Happy Fluttering!
-
-[issues]: https://github.com/flutter/flutter/issues
-[PRs]: https://github.com/flutter/flutter/pulls
-[discord]: https://discord.gg/rflutterdev
-[doc-issues]: https://github.com/flutter/website/issues
-[doc-PRs]: https://github.com/flutter/website/pulls
-[so]: https://stackoverflow.com/tags/flutter
-[mailinglist]: https://groups.google.com/d/forum/flutter-dev
-[chat]: https://github.com/flutter/flutter/wiki/Chat
-[reddit]: https://www.reddit.com/r/FlutterDev
+If you would like to contribute to the Flutter website,
+we’re happy to have your help! Anyone can contribute,
+regardless of your writing experience or how you view yourself as a writer,
+developer, or grammar stickler.
+
+## Contributors
+
+Before submitting a PR,
+[search for an issue][search-issue] that covers your concern.
+If you don't find one and your change is significant,
+consider [creating an issue][create-issue]
+that describers the problem and how you would plan to fix it.
+
+When you create a new page or perform a major rewrite of an existing page,
+consider drafting it first in Google Docs to simplify collaboration.
+
+### Draft changes
+
+1. [Create a branch][create-branch] for your change.
+   Consider naming it `fix-<issue_number>` to help others find the related issue.
+   The new branch should be based on the `main` or a feature or version branch.
+
+1. Try to limit content changes to those that address the request made in the
+   associated issue or doc plan. Use your best judgment.
+
+   1. Avoid adjusting indents, line breaks, formatting, and the like
+      unrelated to your PR as part of the PR. Separate different work
+      using a new commit, a separate issue, a separate PR, or some
```

Comment:
Would "a new commit" still be in the same PR? The previous sentence says "as part of the PR"